### DISCUSSION TITLE
Sequelize belongsTo called with something that's not a subclass of sequelize.model

### DISCUSSION THREAD

**Discussion by ilomon10** [feathersjs/feathers#2351] [Q&A]
### Steps to reproduce

1. Generate service
2. Get script like this
```ts
// eslint-disable-next-line @typescript-eslint/no-unused-vars
(devices as any).associate = function (models: any): void {
    // Define associations here
    // See http://docs.sequelizejs.com/en/latest/docs/associations/
};
```
3. Add some associate
```ts
// eslint-disable-next-line @typescript-eslint/no-unused-vars
(devices as any).associate = function (models: any): void {
    // Define associations here
    // See http://docs.sequelizejs.com/en/latest/docs/associations/
    const { users } = models;
    devices.belongsTo(users);
};
```
4. Run script `npm run dev`
```bash
$ ts-node-dev --no-notify src/
```

Note:
Working fine when run script `npm run compile`
```bash
$ shx rm -rf lib/ && tsc
```

### Expected behavior
Working fine just like before

### Actual behavior
Showing error
```
Error: devices.belongsTo called with something that's not a subclass of Sequelize.Model
    at Function.<anonymous> (PATH/node_modules/sequelize/lib/associations/mixin.js:93:13)
    at Function.devices.associate (PATH/src/models/devices.model.ts:49:13)
    at PATH/src/sequelize.ts:27:31
    at Array.forEach (<anonymous>)
    at Function.app.setup (PATH/src/sequelize.ts:25:25)
    at Function.listen (PATH/node_modules/@feathersjs/express/lib/index.js:63:12)
    at Function.newMethod (PATH/node_modules/uberproto/lib/proto.js:34:20)
    at Function.listen (PATH/node_modules/@feathersjs/socketio/lib/index.js:39:25)
    at Function.newMethod [as listen] (PATH/node_modules/uberproto/lib/proto.js:34:20)
    at Object.<anonymous> (PATH/src/index.ts:5:20)
```
And more random error like `app.service("SOME_SERVICE") Property "patch" does not exist on the "never"`

### System configuration

**Module versions**:
@feathersjs/errors: `^4.5.8`
@feathersjs/express: `^4.5.8`
@feathersjs/feathers: `^4.5.8`
feathers-sequelize: `^6.2.0`

**NodeJS version**: `14.16.1`

**Operating System**: `Debian 10 WSL` on Windows 10

**Module Loader**: `ts-node-dev`

**Comment by ilomon10** [comment]
WHYYYYYY!!!!!
![image](https://user-images.githubusercontent.com/25215779/120591342-90387d80-c46e-11eb-80e0-f2115de8bd8a.png)

When i remove that line 58 no error
![image](https://user-images.githubusercontent.com/25215779/120591401-ab0af200-c46e-11eb-99ae-0c7e92ab4a12.png)

**Reply by JSegundo** [reply]
please tell me you find the answer! i dont know why this happens.

**Reply by oendnsk675** [reply]
fk code, just a simple relationship, but make bruhhhh 🔥 confusion

**Comment by JSegundo** [comment]
https://stackoverflow.com/questions/61709290/sequelize-model-association-wont-create-a-new-column/61710568#61710568

**Comment by muhammadullah1** [comment]
its solve my problem  const { users } = models; the models.users error me
